subsys/sal/sid_pal/src/sid_ble_adapter.c

```diff
@@ -404,8 +438,10 @@ static sid_error_t ble_adapter_deinit(void)
 {
 	LOG_DBG("Sidewalk -> BLE");
 	sid_ble_conn_deinit();
-
-	int err = bt_disable();
+	sid_ble_advert_deinit();
```

Comment:
Why to add advertisiement deinit here?

Reply:
Because we can free the advertising from BLE stack. Maybe for now it is not very useful, but I figured out if we asked for some resources, it would be nice to give them back when we are done working with them.